yb-voyager/src/queryparser/helpers.go

```diff
@@ -94,9 +105,11 @@ func GetStringValueFromNode(nodeMsg protoreflect.Message) string {
 	nodeType := node.Descriptor().FullName()
 	switch nodeType {
 	case PG_QUERY_STRING_NODE:
-		strField := node.Descriptor().Fields().ByName("sval")
-		strValue := node.Get(strField)
-		return strValue.String()
+		return extractStringField(node, "sval")
```

Comment:
can you add one-liner comments here explaining what these node types are?

Reply:
done